Formula/pandoc.rb

```diff
@@ -22,11 +22,6 @@ class Pandoc < Formula
   depends_on "gmp"
```

Comment:
Looks like it's a runtime dep, according to https://github.com/jgm/pandoc/issues/2943#issuecomment-221423238